department-of-veterans-affairs/caseflow
# Adding indicator enhancement to "What's New?" page. closes #553

# Blocked by #554. Please merge that PR first.

Best (manual) way to test:
1. Change the `date-last-updated` attribute in `updates.html.erb`. Use dates in the past or future.
2. Load a page in Caseflow (while logged in).
3. See whether there's a star next to "What's New?" in the menu. 
4. Visit the "What's New?" page. Repeat step 3.

To repeat the test, delete the `caseflowupdatesviewed` key from `localStorage`.

Passes: 
1. If you see the star and the value of `date-last-updated` is in the past.
2. If `date-last-updated` is in the future, you should not see a star.
3. If you no longer see a star after visiting the "What's New?" page.
4. If no network requests occur on subsequent pages after you visit the "What's New?" page (open the network console to verify.)
